xc/xenctrlext_stubs.c

```diff
@@ -195,6 +196,29 @@ CAMLprim value stub_xenctrlext_domain_set_target(value xch,
 	CAMLreturn(Val_unit);
 }
 
+CAMLprim value stub_xenctrlext_domain_vcpu_hotplug(value xch, value domid, value ncpus)
+{
+	CAMLparam2(domid, ncpus);
+
+	int ret, i, bytes;
+	uint8_t *map;
+
+	bytes = (_D(ncpus) + 7) / 8;
+	map = calloc(bytes, 1);
+	if (!map)
+		uerror("calloc fail", Nothing);
+
+	for (i = 0; i < _D(ncpus); i++)
```

Comment:
This looks correct to me but a bit inefficient. The code constructs a map with ncpu bits set to 1 and this is done bit by bit. However, I would keep this because I believe it is correct and I expect ncpus to be small.

Reply:
Thanks, so won't change here.